src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Compile.cs

```diff
@@ -65,9 +66,13 @@ void ProcessDirectory (ITaskItem item, object lockObject)
 			string fileOrDirectory = item.GetMetadata ("ResourceDirectory");
 			if (string.IsNullOrEmpty (fileOrDirectory) || !isDirectory)
 				fileOrDirectory = item.ItemSpec;
+			if (isDirectory && !Directory.Exists (fileOrDirectory)) {
+				LogWarning ($"Ignoring '{fileOrDirectory}' as it does not exist!");
```

Comment:
Should this say `Ignoring directory …`, so that it's more explicit that it's a directory which doesn't exist, not a file?